test/ngAria/ariaSpec.js

```diff
@@ -640,26 +665,6 @@ describe('$aria', function() {
       compileElement('<div ng-dblclick="someAction()" tabindex="userSetValue"></div>');
       expect(element.attr('tabindex')).toBe('userSetValue');
     });
-
-    it('should set proper tabindex values for radiogroup', function() {
```

Comment:
I decided Angular shouldn't be in the business of fixing custom radiogroups...it's too big of a can of worms when it comes to expected keyboard behavior.